docs/application/dotnet/guides/security/tee-client.md

```diff
@@ -1,5 +1,7 @@
 # TEE Communication
 
+> **Note**   
+> Tizen.Security.TEEC is deprecated since Tizen 6.5 and will be removed after two releases without any alternatives.
```

Comment:
Since this change is for 6.5 kindly raise this PR for tizen_6.5_prepare.

Tizen.Security.TEEC in code tag?

Reply:
Done.